src/utils/filteredTodos.ts

```diff
@@ -0,0 +1,18 @@
+import { Actions } from '../types/Actions';
+import { Todo } from '../types/Todo';
+
+export const filteredTodos = (todos: Todo[], action: Actions): Todo[] => {
```

Comment:
```suggestion
export const getFilteredTodos = (todos: Todo[], action: Actions): Todo[] => {
```
it's function, not already filtered todos